row_cache.hh

```diff
@@ -381,14 +395,18 @@ public:
                                      reader_permit permit,
                                      const dht::partition_range&,
                                      const query::partition_slice&,
+                                     const tombstone_gc_state*,
                                      const io_priority_class& = default_priority_class(),
                                      tracing::trace_state_ptr trace_state = nullptr,
                                      streamed_mutation::forwarding fwd = streamed_mutation::forwarding::no,
                                      mutation_reader::forwarding fwd_mr = mutation_reader::forwarding::no);
 
-    flat_mutation_reader_v2 make_reader(schema_ptr s, reader_permit permit, const dht::partition_range& range = query::full_partition_range) {
+    flat_mutation_reader_v2 make_reader(schema_ptr s,
+                                    reader_permit permit,
+                                    const dht::partition_range& range = query::full_partition_range,
+                                    const tombstone_gc_state* gc_state = nullptr) {
         auto& full_slice = s->full_slice();
-        return make_reader(std::move(s), std::move(permit), range, full_slice);
+        return make_reader(std::move(s), std::move(permit), range, full_slice, gc_state);
```

Comment:
Unit test which demonstrates this feature in row_cache_test.cc is missing.

Reply:
Thanks, will do